### ISSUE TITLE
easylist.club: badware

### ISSUE THREAD

**Issue by Sonic5435325** [uBlockOrigin/uAssets#18353]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

Just a redirect site to another site blocked by uBlock Origin, just like the last one I reported.

### URL(s) where the issue occurs.

`https://easylist.club`

### uBO version

v1.49.3rc15

### Browser name and version

Firefox 113.0.2

### Settings

- Use custom filters, check "I am an advanced user"

**Comment by stephenhawk8054** [comment]
That domain is parked.

**Comment by Sonic5435325** [comment]
@stephenhawk8054 This is what I see
![Screenshot from 2023-06-01 21-50-24](https://github.com/uBlockOrigin/uAssets/assets/89158249/8179596a-81e1-4c40-b855-6c934a1202b4)
![Screenshot from 2023-06-01 21-51-04](https://github.com/uBlockOrigin/uAssets/assets/89158249/dc0d622a-2658-4fdd-82e2-92e39d31abab)
